lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_poland_when_residing_in_uk.govspeak.erb

```diff
@@ -48,12 +48,7 @@
 
   You will be asked to provide a copy of your birth certificate. Make sure you send a [certified copy](/order-copy-birth-death-marriage-certificate) as you won’t get it back.
 
-  <%= render partial: 'consular_fees_table.govspeak.erb',
-             locals: { calculator: calculator } %>
-  <%= render partial: 'link_to_consular_fees.govspeak.erb',
-             locals: { calculator: calculator } %>
-
-  <%= render partial: 'payment_information.govspeak.erb',
+  <%= render partial: 'os_affirmation/fees.govspeak.erb',
```

Comment:
I've updated the commit note to make it clearer that it's changing two outcomes.